datasets/us/nd/med_exclusions/crawler.py

```diff
@@ -0,0 +1,69 @@
+from typing import Dict
+from rigour.mime.types import XLSX
+from openpyxl import load_workbook
+
+from zavod import Context, helpers as h
+
+
+def crawl_item(row: Dict[str, str], context: Context):
+
+    if not row.get("provider_name"):
+        return
+
+    provider_name = row.pop("provider_name")
```

Comment:
I don't think the owner split is simple because of cases like this:

![Screenshot 2024-09-12 at 15 53 43](https://github.com/user-attachments/assets/fab56c10-0021-4b83-85ff-1f0cd747aada)

This looks like three first names

Reply:
For the alias I was able to do the code.

Reply:
ah wel spotted. Ok. I think it'd be good to clean up the company names though. Let's just h.multisplit on `["owners:", "owner:"]` and if there's a second part, put it in entity description.